nectar/downloaders/threaded.py

```diff
@@ -100,18 +98,21 @@ def worker(self, queue):
         """
         try:
             session = build_session(self.config)
-
             while True:
                 request = queue.get()
-
                 if request is None or self.is_canceled:
                     session.close()
                     break
-
-                self._fetch(request, session)
+                for retry in xrange(self.retries):
```

Comment:
The way this is implemented, I think the term "retries" is misleading. With the default value of "3", I think nectar will try the request up to 3 times in total. One initial try, and then two retries. If I set "retries" to 0, I expect the request to still happen, but to never retry.

I suggested above adding a doc block because putting the expected behavior into words will I think help clarify what the "right thing to do" is.